src/test/compile-fail/issue-60989.rs

```diff
@@ -0,0 +1,4 @@
+fn main() {
+    let c1 = ();
+    c1::<()>;
```

Comment:
We might also want to extend this with the other forms in the issue?

Reply:
will do on master. This is just for stable.

Reply:
Yeah good idea 👍 -- Let's leave this here as a reminder.